resource-managers/kubernetes/integration-tests/dev/dev-run-integration-tests.sh

```diff
@@ -103,4 +104,4 @@ then
   properties=( ${properties[@]} -Dtest.exclude.tags=$EXCLUDE_TAGS )
 fi
 
-$TEST_ROOT_DIR/build/mvn integration-test -f $TEST_ROOT_DIR/pom.xml -pl resource-managers/kubernetes/integration-tests -am -Pkubernetes ${properties[@]}
+$TEST_ROOT_DIR/build/mvn integration-test -f $TEST_ROOT_DIR/pom.xml -pl resource-managers/kubernetes/integration-tests -am -Pscala-$SCALA_VERSION -Pkubernetes-integration-tests ${properties[@]}
```

Comment:
@skonto don't you need `-Pkubernetes` here too?
Otherwise, I don't even think we need to wait for another test. The exact POM change above just passed in the other PR and this part won't be tested anyway. I tested the SCALA_VERSION evaluation manually and it works.

Reply:
Yes it is needed since kubernetes is not a module of the tests profile. I updated the PR.